source/2020-04-03-the-ember-times-issue-142.md

```diff
@@ -61,14 +62,14 @@ Try it out today by ember installing [ember-arg-types](https://github.com/jkusa/
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Betas of Glimmer.js 2 🎉](https://github.com/glimmerjs/glimmer.js/releases)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+A set of **new betas for Glimmer.js 2.0.0** ([beta.1](https://github.com/glimmerjs/glimmer.js/releases/tag/v2.0.0-beta.1) && [beta.3](https://github.com/glimmerjs/glimmer.js/releases/tag/v2.0.0-beta.3)) have been released! 🔥🔥🔥 
+
+The releases incorporate some exciting new developments including a new minimal `renderComponent` API, a template import based design, a standardized build based on plain JavaScript modules, a new Glimmer.js app blueprint and support for custom helpers and modifiers. The standardized build unlocks the ability to use whatever custom build pipeline you might want (WebPack, Parcel, Rollup, etc.)!
```

Comment:
Because there are several examples of new developments, I think adding a sentence before the last example will help with reading.
```suggestion
The releases incorporate some exciting new developments including a new minimal `renderComponent` API, a template import based design, a standardized build based on plain JavaScript modules, a new Glimmer.js app blueprint, and support for custom helpers and modifiers. The standardized build unlocks the ability to use whatever custom build pipeline you might want (WebPack, Parcel, Rollup, etc.)!
```